tasks/install/Solaris.yml

```diff
@@ -0,0 +1,37 @@
+---
+
+- name: filebeat | Create Solaris log directory and parents
+  file:
+    path: "{{ filebeat_log_path }}"
+    owner: "{{ filebeat_user }}"
+    group: "{{ filebeat_group }}"
+    state: directory
+    recurse: yes
+    mode: 0775
+
+- name: filebeat | Download Filebeat agent package
+  get_url:
+    url: "{{ filebeat_solaris_url }}"
+    dest: "/tmp/filebeat-{{ filebeat_version }}.pkg"
+    mode: 0550
+
+- name:  filebeat | Install Filebeat agent for Solaris
+  svr4pkg:
+    name: filebeat
+    src: "/tmp/filebeat-{{ filebeat_version }}.pkg"
+    state: present
+
+- name: filebeat | Install Filebeat service
+  shell: (svccfg import /opt/filebeat/etc/filebeat.xml) && (svcs | grep filebeat) > /opt/filebeat/service.created
+  args:
+    creates: /opt/filebeat/service.created
+  when: filebeat_start_at_boot
+
+- name: filebeat | Ensure service is running
```

Comment:
Thanks for your review. main.yml is the better location. but:
```
- service:
    name: filebeat
    enabled: yes
  when: filebeat_start_at_boot
```
and 
```
- name: filebeat | Ensure service is running
  service:
    name: filebeat
    state: started
  when: filebeat_start_at_boot
```
to my understanding, are not the same. In main.yml we are just ensuring it is enabled, but not started.

sounds like a good solution would be removing my service section and adding `state: started` to main.yml, if you don't mind.